virttest/remote_commander/remote_runner.py

```diff
@@ -295,6 +297,10 @@ def parse_func_name(self, func_name, commander):
         :param func_name: Function name
         :param commander: Where to execute the command (remote or local)
         """
+        for name in func_name:
+            if name == "run_test":
+                # Always run external test in the current process.
+                self.manage = True
```

Comment:
Yeah, I understand and agree with you. I'd just raise an exception when the user tries to do it instead of silently changing the behavior.